src/test/java/api/requests/scenarios/HoldShelfExpirationDateTests.java

```diff
@@ -93,16 +98,55 @@ void requestWithShelfExpirationDateForSpExpiryInHoursAndMinutes(
     assertThat("request status snapshot in storage is " + OPEN_AWAITING_PICKUP,
       storedRequest.getString("status"), is(OPEN_AWAITING_PICKUP));
 
-    assertThat("request hold shelf expiration date is " + amount + " " + interval.toString() + " in the future",
+    ZonedDateTime zdtWithZoneOffset  = ZonedDateTime.parse(storedRequest.getString("holdShelfExpirationDate"),
+      DateTimeFormatter.ISO_ZONED_DATE_TIME);
+
+    if(interval.addTo(getZonedDateTime(), amount).toLocalDate().equals(LocalDate.now(zdtWithZoneOffset.getZone()))){
```

Comment:
Spaces are needed after the `if` and before the `{`.

Also could the expression `interval.addTo(getZonedDateTime(), amount).toLocalDate().equals(LocalDate.now(zdtWithZoneOffset.getZone()))` be placed into a method with an name starting with `is...` so that it is more clear what this condition is checking for?

Reply:
added